index.js

```diff
@@ -212,24 +241,58 @@ NYC.prototype.report = function (cb, _collector, _reporter) {
   reporter.write(collector, true, cb)
 }
 
+var loadedMaps = {}
+
 NYC.prototype._loadReports = function () {
   var _this = this
   var files = fs.readdirSync(this.tmpDirectory())
 
-  return _.map(files, function (f) {
+  var SourceMapCache = require('./lib/source-map-cache')
```

Comment:
Did you move the require statement down here because `source-map-cache` pulls in `lodash`? Because we could probably remove it there as well.